TUM Campus App/NewsTableViewController.swift

```diff
@@ -37,6 +37,7 @@ extension NewsTableViewController {
         tableView.estimatedRowHeight = tableView.rowHeight
         tableView.rowHeight = UITableViewAutomaticDimension
         tableView.tableFooterView = UIView(frame: CGRect.zero)
+        tableView.scrollToRow(at: IndexPath(row: getNextUpcomingNewsRow(), section: 0), at: UITableViewScrollPosition.top, animated: false)
```

Comment:
Well, I thought the animation was to fast when there are a a couple of news item before the current one. Since scrollToRow is called in viewDidLoad (and not viewDidAppear), the user wont see the list jumping, but directly sees the wished story on top of the screen.
Personally I'd keep it like this.